### DISCUSSION TITLE
2023-08-09

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#295] [daily_meeting]
### 오늘의 할 일
마코,테오 - OAuth 로그인 기능 마무리
히이로, 모디 - Logback 로그 내용 프로메테우스를 통해 전송하는 방법 찾기
프론트 - OAuth 페이지 관련 개발

### 언젠가 할 일
소켓, 폴링 양방향 통신 어떻게 할지 (4차 스프린트 내에 그냥 진행하고 발표만 안하는 방향으로 갈건지 로그인 이후 결정하기)
질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
브랜치 전략 결정하기(dev -> main)
A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)
### 개인 일정
히이로 1시 면담
노아 5시 근로

**Comment by nlom0218** [comment]
### 체크인

10점 - 옥상에 올라가면 날씨가 은근 선선하네요...? 그리고 올만에 도시락을 가져왔습니담!

### 체크아웃

10점 - 로그인 페이지 같이 만드니 재밌네요!

**Comment by woosung1223** [comment]
### 체크인
9점 - 졸리긴 하지만 아침에 커피 사갖고 와서 다행입니다..

### 체크아웃
10점 - 오늘 인증 기능 완료해서 기분이 좋습니다

**Comment by woo-jk** [comment]
### 체크인
8점 - 피곤해서 -1, 어제 집공부를 못해서 -1

### 체크아웃
10점 - 내일 재택이라 ㅎㅎ

**Comment by jaehee329** [comment]
### 체크인
8점 - 조오금 시원한 아침이어서 어제 아침보다 +1

### 체크아웃
7점 - 프롬테일 + 로키 설정이 아직 안 끝나서 -1

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 아침에 너무 더워서 찝찝하게 등교해서 컨디션이 별로였는데 오늘 드디어 방에 에어컨이 들어오기 때문에 9점으로 하루를 시작해봅니다.

### 체크아웃
9.5점 - 내일 재택이라 기분이 좋아졌습니다. 오늘 하기로 한 일만 잘 끝내고 가면 좋겠네요!

**Comment by aak2075** [comment]
### 체크인

10점 - 날씨도 시원하고 간만에 컴포즈 모바일 주문이 열려서 컴포즈 마셔서 좋네요

### 체크아웃

10점 - OAuth 구현 완료.

**Comment by yeopto** [comment]
### 체크인

8점 - 어제 농구하고 푹 잤는데 몸이 찌뿌둥해서 -2

### 체크아웃

10점 - 내일 재택이라 너무 좋습니다..
